faq/how.md

```diff
@@ -0,0 +1,13 @@
+---
+title: How does SCN Work?
+parent: FAQ
+nav_order: 4
+---
+
+# How Does the Seattle Community Network Work?
+
+The Seattle Community Network partners with the University of Washington to share free or low-cost internet access with areas of higher need.
+
+The Seattle Community Network functions by utilizing a portion of the University of Washington's internet bandwidth and sharing that access wirelessly with certain areas covered by the network. This is accomplished by using high gain wireless antennas to transmit data from the hardwired connection to a local node, which is then shared to nearby devices using the LTE (cell-phone data) standard. Individual users can then connect to this signal using an SCN-provided hardware, creating a local WiFi network to use for internet access in their home.
+
+The network is completely created, managed, and maintained by volunteers with a range of diverse skills in information technology and beyond. All infrastructure is paid for by generous donations from sponsors and the public. Speaking of which, why not VOLUNTEER or DONATE?
```

Comment:
let's make "volunteer" and "donate" links rather than all caps!

Reply:
Change completed in 849ddb8.